test/integration/controllers/lesson.spec.js

```diff
@@ -62,4 +62,19 @@ describe('Lesson controller', () => {
       expectError(401, UNAUTHORIZED, response)
     })
   })
+
+  describe(`GET ${endpointUrl}`, () => {
+    it('get all lessons', async () => {
+      const response = await app.get(endpointUrl).set('Authorization', `Bearer ${accessToken}`)
+
+      expect(response.status).toBe(200)
+      expect(response.body).toEqual(expect.objectContaining({ count: 1, items: [expect.any(Object)] }))
+    })
+
+    it('should throw UNAUTHORIZED', async () => {
+      const response = await app.get(endpointUrl)
+
+      expectError(401, UNAUTHORIZED, response)
+    })
+  })
```

Comment:
i think it would be good to add case with forbidden error case

Reply:
u r right! fixed it